featuretools/primitives/base/primitive_base.py

```diff
@@ -33,6 +33,8 @@ class PrimitiveBase(object):
     commutative = False
     # (bool) If True, is compatible with Dask EntitySets
     dask_compatible = False
+    # (bool) If True, is compatible with Koalas EntitySets
+    koalas_compatible = False
```

Comment:
what if we refactored this to be `compatible_dataframes = [pd.DataFrame, ks.DataFrame, dd.DataFrame]` or something like that? might be a more scalable approach as we add other dataframe options like cudf

Reply:
that could be done in a PR separate from this one. maybe make it first, if it would make this one a bit smaller